src/server/services/metadata.ts

```diff
@@ -15,274 +15,302 @@ export const legalPracticeAreasList = [
 ];
 
 export const countriesList = [
-  { text: "Afghanistan", value: "Afghanistan" },
-  { text: "Albania", value: "Albania" },
-  { text: "Algeria", value: "Algeria" },
-  { text: "American Samoa", value: "American Samoa" },
-  { text: "Andorra", value: "Andorra" },
-  { text: "Angola", value: "Angola" },
-  { text: "Anguilla", value: "Anguilla" },
-  { text: "Antigua and Barbuda", value: "Antigua and Barbuda" },
-  { text: "Argentina", value: "Argentina" },
-  { text: "Armenia", value: "Armenia" },
-  { text: "Aruba", value: "Aruba" },
-  { text: "Australia", value: "Australia" },
-  { text: "Austria", value: "Austria" },
-  { text: "Azerbaijan", value: "Azerbaijan" },
-  { text: "Bahamas", value: "Bahamas" },
-  { text: "Bahrain", value: "Bahrain" },
-  { text: "Bangladesh", value: "Bangladesh" },
-  { text: "Barbados", value: "Barbados" },
-  { text: "Belarus", value: "Belarus" },
-  { text: "Belgium", value: "Belgium" },
-  { text: "Belize", value: "Belize" },
-  { text: "Benin", value: "Benin" },
-  { text: "Bermuda", value: "Bermuda" },
-  { text: "Bhutan", value: "Bhutan" },
-  { text: "Bolivia", value: "Bolivia" },
+  { text: "Afghanistan", value: "Afghanistan", code: "AFG" },
```

Comment:
yes, I think we should have a separate card to standardise the use of countries so they're driven off ISO country code rather than country name.  This will insulate us from if they choose to update the name in future so it won't impact any functionality or data, e.g., changing Swaziland to Eswatini

Reply:
Ah I didn't know the runner had 3 character ISO codes. Are there any specific changes that should be made to this ticket because of this change?

Reply:
Currently the runner doesn't send 3 character ISO country codes but Jen's suggesting that maybe we should support this.  I'm suggesting we address that with a separate card rather than integrate into this one as they address separate issues and it will require more widespread changes to the form runner, deserialisers and the find service to drive countries off the country code rather than the country name as we do today.  As this is just a back-end tech task I'll draw something up to cover it separately.